### PR TITLE
test: update sdk-test-data

### PR DESCRIPTION
**Pull request by rasendubi**

### CODE REVIEW

**Comment by rasendubi** [comment]
@aarsilv example of `sdk-test-data` update PR ^

Nice thing I didn't know is that "Files changed" tab allows you to navigate to the diff in the submodule, so I can review what changes were applied to `sdk-test-data`, so I can pinpoint the issue more easily.

**Comment by rasendubi** [comment]
cc @typotter

File: eppo_core/src/eval/eval_bandits.rs [hunk 1]
```diff
@@ -265,8 +265,11 @@ impl BanditModelData {
                     // In the case of multiple actions getting the same best score, we need to break
                     // the tie deterministically.
                     //
-                    // Compare action names next
-                    Ord::cmp(a.0, b.0)
+                    // Compare action names next.
+                    //
+                    // We're reversing the comparison, so that before-ordered name is considered
+                    // higher and wins the best score.
+                    Ord::cmp(a.0, b.0).reverse()
```
Comment (rasendubi) [inline eppo_core/src/eval/eval_bandits.rs:272]: We already compared by name but picking the name later in the order

**Comment by rasendubi** [comment]
This PR also includes pin to https://github.com/Eppo-exp/sdk-test-data/pull/54

**Comment by aarsilv** [comment]
Neato! You may be converting me... 😅

File: eppo_core/src/eval/eval_bandits.rs (hunk 1 above)
Comment (aarsilv) [reply eppo_core/src/eval/eval_bandits.rs:272]: Ah do we need the `reverse()` because of the surrounding (`max_by()`)?

**Review by aarsilv — APPROVED** [review]

File: eppo_core/src/eval/eval_bandits.rs (hunk 1 above)
Comment (rasendubi) [reply eppo_core/src/eval/eval_bandits.rs:272]: Yes. `max_by()` selects the highest value, so with `"a" < "b"` it was selecting the "b"

**Pull request merged**